apps/server/src/index.ts

```diff
@@ -1,16 +1,21 @@
 import bodyParser from 'body-parser'
 import express from 'express'
 import cors from 'cors'
+import { productRouter } from './routes/products'
+import { logErrors } from './middlewares/logErrorMiddleware'
+import { healthchecker } from './routes/health/helthchecker'
+import { healthcheckerMiddleware } from './middlewares/helthcheckerMiddleware'
 
 const app = express()
 
 app.use(bodyParser.urlencoded({ extended: true }))
 app.use(bodyParser.json())
 app.use(cors())
 
-app.get('/', (_, res) => {
-  return res.json({ ok: true })
-})
+app.use(healthcheckerMiddleware)
+app.use('/api-v1', productRouter)
+app.use('/api-v1', healthchecker)
```

Comment:
Great strategy for API versioning

Reply:
Thank, Learned that in my previous  job.